NanoProd/crab/Run3_2022/VV.yaml

```diff
@@ -28,3 +28,13 @@ ZZto2Nu2Q_powheg: /ZZto2Nu2Q_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv
 ZZto2Nu2Q_powheg_ext1: /ZZto2Nu2Q_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5_ext1-v2/MINIAODSIM
 ZZto4L_powheg: /ZZto4L_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
 ZZto4L_powheg_ext1: /ZZto4L_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5_ext1-v2/MINIAODSIM
+ZZto2L2Q_amcatnloFXFX: /ZZto2L2Q-1Jets_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
```

Comment:
Remove all di-boson amcatnlo samples. Only powheg (or VV inclusive) should be used.